rktlet/image/imagestore.go

```diff
@@ -77,49 +80,71 @@ func (s *ImageStore) ImageStatus(ctx context.Context, req *runtime.ImageStatusRe
 		return nil, err
 	}
 
+	reqImg := req.GetImage().GetImage()
```

Comment:
ick, either `GetImage` call could return nil.
Is there any sort of programmatic way for dealing with the always-optional world of protobufs?